charts/sourcegraph/templates/qdrant/qdrant.Service.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.qdrant.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    sourcegraph.prometheus/scrape: "true"
+    prometheus.io/port: "6333"
```

Comment:
I tested that these populate metrics in Grafana. I'll need to add a basic dashboard for qdrant